src/proxy.sol

```diff
@@ -16,37 +16,65 @@
 
 pragma solidity >=0.6.0 <0.8.0;
 
-import { Title, TitleOwned, TitleLike } from "tinlake-title/title.sol";
+import "tinlake-auth/auth.sol";
 
 interface RegistryLike {
     function cacheRead(bytes memory _code) external view returns (address);
     function cacheWrite(bytes memory _code) external returns (address target);
 }
-// Proxy is a proxy contract that is controlled by a Title NFT (see tinlake-title)
-// The proxy execute methods are copied from ds-proxy/src/proxy.sol:DSProxy
-// (see https://github.com/dapphub/ds-proxy)
-contract Proxy is TitleOwned {
-    uint public         accessToken;
+
+contract Proxy is Auth {
+
+    mapping (address => uint256) public users;
```

Comment:
I think it's a bit confusing that `users` are explicitly included, while `wards` are imported. I'd remove the dependency on `tinlake-auth` and just add `wards` inline like in https://github.com/centrifuge/connectors/blob/main/src/Connector.sol